### PR TITLE
laptop: Changing shells should be opt-in only

### PR DESCRIPTION
**Pull request by chub**
Changing the shell to zsh should not be opt-out.  If someone cares about using
an alternative shell (than bash), they'll know how to change it.

### CODE REVIEW

**Comment by chub** [comment]
I tested the script by running it with default arguments, and verified that the "things to add to your shellrc" was omitted.

This might be a good time to consider appending it to `.profile` or `.bash_profile`, whichever works on macOS.

File: mac [hunk 1]
```diff
@@ -259,15 +174,11 @@ if [ -f "$HOME/.laptop.local" ]; then
...
 fancy_echo "Script complete!"
-
-if [[ "${SKIP_ZSH}" == 1 ]]; then
-  fancy_echo "Since zsh was not installed, please add the following to your shell rc:"
-  echo ""
-  for line in "${SHELL_CHANGES[@]}"; do
+fancy_echo ""
+fancy_echo "=============================================="
+fancy_echo ""
+fancy_echo "Please add the following to your shell rc:"
```
Comment (nathandcornell) [inline mac]: Should we add a couple example filenames just in case this isn't clear?
e.g.
```
... to your shell rc (.zshrc, .bashrc, etc)
```

**Review by nathandcornell — APPROVED** [review]
Looks great, sorry for the late review!!
Just one non-blocking suggestion

**Comment by chub** [comment]
Thanks for the review!

**Pull request merged**